tests/src/test/java/alluxio/master/file/ConcurrentFileSystemMasterSetTtlTest.java

```diff
@@ -103,18 +88,20 @@ public void rootFileConcurrentSetTtlTest() throws Exception {
     final int numThreads = CONCURRENCY_FACTOR;
     AlluxioURI[] files = new AlluxioURI[numThreads];
     long[] ttls = new long[numThreads];
+    Random random = new Random();
 
     for (int i = 0; i < numThreads; i++) {
       files[i] = new AlluxioURI("/file" + i);
       mFileSystem.createFile(files[i],
           CreateFileOptions.defaults().setWriteType(WriteType.MUST_CACHE)).close();
-      ttls[i] = 1L + (long) (Math.random() * 1000);
+      ttls[i] = random.nextInt(TTL_INTERVAL_MS);
```

Comment:
Previously the test had all TTLs > the TTL interval, I think we should make the range so that both cases are tested (> and < interval)